ccloud/ccloud-generate-cp-configs.sh

```diff
@@ -59,20 +59,30 @@ set -eu
 
 ################################################################################
 # Confluent Cloud configuration
+#
+# Example file at $HOME/.ccloud/config
+#
+#   $ cat $HOME/.ccloud/config
+#   bootstrap.servers=<BROKER ENDPOINT>
+#   ssl.endpoint.identification.algorithm=https
+#   security.protocol=SASL_SSL
+#   sasl.mechanism=PLAIN
+#   sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username\="<API KEY>" password\="<API SECRET>";
+#
```

Comment:
I think moving this into the README might help a user

Reply:
Added into the README at https://github.com/confluentinc/examples/pull/208/commits/7312967a5cd9ba7b6c267a37eacdcd9e646f36ed

I also have a separate PR waiting merge for docs (will update examples later)